packages/wonder-blocks-color/index.js

```diff
@@ -18,6 +18,7 @@ const Color = {
     offBlack50: fade(offBlack, 0.5),
     offBlack32: fade(offBlack, 0.32),
     offBlack16: fade(offBlack, 0.16),
+    offBlack10: fade(offBlack, 0.1),
```

Comment:
The design had this color as the box-shadow.

Reply:
We shouldn't add this to wonder-blocks-color unless it gets add to the design spec.  In the mean time use `fade(offBlack, 0.1)` directly in the packages you're implementing.